pkg/node/sendoperation/callsc/callsc.go

```diff
@@ -8,7 +8,10 @@ import (
 	utils "github.com/massalabs/station/pkg/node/sendoperation/serializeaddress"
 )
 
-const CallSCOpID = uint64(4)
+const (
+	CallSCOpID                      = uint64(4)
+	addressWithoutPrefixVersionSize = 32
```

Comment:
maybe name it publicKeyHashSize? because it's address without type and version prefix actually.